layers/+emacs/emacs-info/packages.el

```diff
@@ -0,0 +1,59 @@
+;;; packages.el --- Org Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst emacs-info-packages
+  '((info :location built-in)))
+
+(defun emacs-info/init-info ()
+  (use-package info
+    :defer t
+    :init
+    (progn
+      (spacemacs|define-transient-state emacs-info
+        :title "Emacs Info transient state"
+        :foreign-keys run
+        :doc
+"
+ [_q_] quit "
+        :bindings
+        ("n" Info-next)
+        ("p" Info-prev)
+        ("e" Info-scroll-up)
+        ("y" Info-scroll-down)
+        ("<tab>" Info-next-reference)
+        ("S-<tab>" Info-prev-reference)
+        ("h" Info-help)
+        ("l" Info-history-back)
+        ("r" Info-history-forward)
+        ("d" Info-directory)
+        ("u" Info-up)
+        ("L" Info-history)
+        ("s" Info-search)
+        ("S" Info-search-case-sensitively)
+        ("i" Info-index)
+        ("I" Info-virtual-index)
+        ("a" info-apropos)
+        ("m" Info-menu)
+        ("gd" Info-goto-node)
+        ("gm" Info-menu)
+        ("gt" Info-top-node)
+        ("gT" Info-toc)
+        ("gf" Info-follow-reference)
+        ("gj" Info-next)
+        ("gk" Info-prev)
+
+        ("q" nil :exit t)
+        :config
+        (evilified-state-evilify-map Info-mode-map
+          :mode Info-mode
+          :bindings
+          (kbd "M-SPC" 'spacemacs/emacs-info-transient-state/body)))
+      (define-key Info-mode-map (kbd "M-SPC") 'spacemacs/emacs-info-transient-state/body))))
```

Comment:
You should also bind your transient state to `s-M-SPC` for all where `M-SPC` is bound by the OS.
```elisp
      (define-key helm-map (kbd "M-SPC")
        'spacemacs/helm-navigation-transient-state/body)
      (define-key helm-map (kbd "s-M-SPC")
        'spacemacs/helm-navigation-transient-state/body)
```

Reply:
What? No, Org-agenda transient is bound to M-SPC. There's some other transient that also binds to M-SPC (I forgot which one).

Reply:
I don't agree there is a difference between being OS agnostic and just being inflexible. A modern emacs distribution should not rely on default keybindings which 90% of users have to manually free from there OS.

Anyway that's just my personal opinion which in fact doesn't matter in this discussion. 

As to your comment, I have double checked the current version of `CONVENTIONS.org` which all spacemacs PRs shall follow. There in the section of `Transient-state` you will find the following paragraph:

```org
* Key bindings conventions
** Reserved prefix
...
...
*** Transient-state
Whenever possible a transient-state should be enabled with ~M-SPC~ and ~s-M-SPC~. We
need the latter bindings on OS X since ~M-SPC~ is used by the OS for spotlight.

For instance transient-states dedicated to special buffers like =helm= or =ido=
buffers are good candidates to be put on ~M-SPC~ and ~s-M-SPC~.

It is recommended to add ~q~ to leave the transient-state.
```
In this paragraph you can find that the requested binding of `s-M-SPC` for the transiant-state is required by the conventions and must be added. You are right not all layers are yet following the latest version of `CONVENTION.org` but this will change in time.